website/src/components/ExpandableGrid/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import styles from "./styles.module.css";
+
+const ExpandableGrid = ({ items, defaultVisibleCount, renderItem }) => {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  const visibleItems = isExpanded ? items : items.slice(0, defaultVisibleCount);
```

Comment:
nits: we could use useMemo here, but it is probably not necessary since this should be a lightweight app and this component won't be rendered for many times.

Reply:
yea lemme keep it simple here for now